Automated Smoketest: :x:
Sha: cc0cf1dcb4f684bdd1b38aa332f1d2dcb40b8c3d
Build: failed
Number of Error(s): 5
make[2]: *** [generated/common.xml] Error 134
make[1]: *** [esp/scm/CMakeFiles/espscm.dir/all] Error 2
[  0%] Building CXX object aws-cpp-sdk-core/CMakeFiles/aws-cpp-sdk-core.dir/source/client/CoreErrors.cpp.o
[100%] Building CXX object aws-cpp-sdk-sqs/CMakeFiles/aws-cpp-sdk-sqs.dir/source/SQSErrors.cpp.o
make: *** [all] Error 2
Get ECLWatch build result

--- a/tools/esdlcomp/esdlcomp.h
+++ b/tools/esdlcomp/esdlcomp.h
@@ -1087,6 +1087,13 @@ public:
             tags = p->next;
             delete p;
         }
+
+        if (name_)
+           free(name_);
+        if (request_)
+           free(request_);
+        if (response_)
+           free(response_);
     }
 
     const char *getName(){return name_;}
@@ -1164,6 +1171,13 @@ public:
         tags=NULL;
         next=NULL;
     }
+    ~EspMountInfo()
+    {
+       if (name_)
+          free(name_);
+       if (localPath_)
+          free(localPath_);
+    }
 
     const char *getName(){return name_;}
     void setName(const char *name)
@@ -1217,6 +1231,12 @@ public:
         name_ =strdup(name);
         tags=NULL;
         next=NULL;
+    }
+
+    ~EspStructInfo()
+    {
+       if (name_)
+          free(name_);
     }
 
     const char *getName(){return name_;}

